native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar-test.android.js

```diff
@@ -73,6 +73,25 @@ describe('android', () => {
       expect(tree).toMatchSnapshot();
     });
 
+    it('should render correctly', () => {
```

Comment:
How about `'should render correctly with custom styles'` so as not to duplicate the name from above?